redos/7/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM centos:7 as prepare
+MAINTAINER Sergei Vorontsov "piligrim@rootnix.net"
+
+ENV REPO_URL=http://mirror.yandex.ru/redos/7.3/x86_64/os/ \
+    CHROOT_DIR="/chroot"
+
+RUN mkdir -p $CHROOT_DIR
+
+RUN for rpm in `curl -s $REPO_URL | grep 'redos-release' | sed 's/<.*>\(.*\)<\/.*>/\1/' | awk '{print $1}'`; do if [[ $rpm =~ redos-release-[0-9].[0-9]-([0-9]|[0-9][0-9]).el7.noarch.rpm ]]; then curl -o $CHROOT_DIR/redos-release.rpm $REPO_URL$rpm; fi; done
+
+#setup base system
+RUN rpm --root=$CHROOT_DIR --rebuilddb
+RUN rpm --root=$CHROOT_DIR --nodeps --quiet -i $CHROOT_DIR/redos-release.rpm
+RUN cp $CHROOT_DIR/etc/pki/rpm-gpg/RPM-GPG-KEY-RED-SOFT /etc/pki/rpm-gpg/RPM-GPG-KEY-RED-SOFT
+
+RUN yum -y --installroot=$CHROOT_DIR update
+
+RUN yum -y --installroot=$CHROOT_DIR install -y \
+    yum \
+    rpm-devel \
+    rpm-devel \
+    devtoolset \
```

Comment:
devtoolset is installed, but not set up? We install and setup devtoolset on CentOS 7 to use GCC 8 instead of default 4.8.5:

https://github.com/packpack/packpack-docker-images/blob/948e9f05366acac80d9d0102c3c5a8dd676240af/centos/7/Dockerfile#L42-L49

But RedOS 7.3 mirror provides GCC 8 on its own.

Reply:
Removed devtoolset.